packages/core/src/browser/widget-decoration.ts

```diff
@@ -97,6 +97,20 @@ export namespace WidgetDecoration {
          */
         readonly fontData?: FontData;
     }
+
+    export namespace TailDecoration {
+        /**
+         * Combines all fields of all `TailDecoration` variants
+         */
+        export interface AnyPartial extends Partial<TailDecoration & TailDecorationIcon & TailDecorationIconClass> { }
```

Comment:
Suggestion: Make this into a type, or is there a special reason for this to be an interface?
```suggestion
        export type AnyPartial = Partial<TailDecoration & TailDecorationIcon & TailDecorationIconClass>;
```